core-services/prow/02_config/_boskos.yaml

```diff
@@ -1,15 +1,378 @@
 # generated with generate-boskos.py; do not edit directly
 resources:
-- max-count: 150
-  min-count: 150
+- names:
+  - us-east-1
```

Comment:
These are the names of the resulting crds, so this will result in exactly one. The boskos resources have a `userdata` field, maybe we can leverage that?

Reply:
Right now we expose the name of the resource via an env var, wie could expose the userdata via another env var.

Reply:
I don't understand how Boskos works.  @stevekuznetsov recommended this "repeat entries in `names`" approach back on 2020-09-10.  He also mentioned userdata and linked [here][1].  I don't really care how I'm supposed to do it, I just want a way to manage per-region leases.  If this approach isn't right, can you spell out what you want in more detail?

[1]: https://github.com/kubernetes-sigs/boskos#adding-userdata-to-a-resource

Reply:
Ok, rerolled to enumerate the repeat entries with 61e9a5ae65 -> ef394fae36.